### PR TITLE
add rules for lebsanft.org

### PR DESCRIPTION
**Pull request by piratenpanda**

### CODE REVIEW

File: src/chrome/content/rules/lebsanft.org.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <target host="*.lebsanft.org" />
+  <securecookie host="^www\.lebsanft\.org$" name=".*" />
+  <exclusion pattern="^http://physik\.lebsanft\.org/"/>
+
+  <test url="http://lebsanft.org/" />
+  <test url="http://physik.lebsanft.org/" />
+  <test url="http://www.lebsanft.org/" />
+
+  <rule from="^http://lebsanft\.org/" to="https://lebsanft.org/"/>
+  <rule from="^http://www\.lebsanft\.org/" to="https://www.lebsanft.org/"/>
```
Comment (reedy) [inline src/chrome/content/rules/lebsanft.org.xml]: You should just use the updated/simplified rule format here

```
<rule from="^http:" to="https:" />
```

You could then just explicitly only target www and no subdomain, and remove the exclusion. Simplies the ruleset somewhat

**Comment by piratenpanda** [comment]
Hopefully this is how it should be

**Comment by reedy** [comment]
I wouldn't have deleted the test urls, nor the secure cookie probably (unless it's unused?).

This ruleset now doesn't secure http://lebsanft.org/ either - is this intended?

**Comment by piratenpanda** [comment]
Oops let me fix that

**Comment by piratenpanda** [comment]
Do I need a rule when the folder without www is just a symlink to www. Will this work for https-everywhere just fine? If yes, then I can just omit it.

**Comment by reedy** [comment]
The server adds the www with a 301, so that's fine

```
chronos@localhost / $ curl -I http://lebsanft.org
HTTP/1.1 301 Moved Permanently
Date: Fri, 17 Apr 2015 13:33:32 GMT
Server: Apache/2.2.15 (CentOS)
X-Powered-By: PHP/5.4.4
X-Pingback: http://www.lebsanft.org/xmlrpc.php
Location: http://www.lebsanft.org/
Connection: close
Content-Type: text/html; charset=UTF-8

chronos@localhost / $
```

**Comment by piratenpanda** [comment]
then I guess this can be merged

**Pull request merged**